bumblebee_status/modules/contrib/shell.py

```diff
@@ -77,8 +78,25 @@ def update(self):
         self.__current_thread.start()
 
     def state(self, _):
-        if self.__output == "no command configured":
-            return "warning"
+        if self.parameter("unit") is None:
```

Comment:
I think this can be shortened by using the "threshold_state()" convenoenxe method that is part of the Module class.